stage1/init/pod_test.go

```diff
@@ -50,8 +50,11 @@ func TestQuoteExec(t *testing.T) {
 			input:  []string{`path with "quo't'es" and \slashes`, `"arg"`, `\`},
 			output: `"path with \"quo\'t\'es\" and \\slashes" "\"arg\"" "\\"`,
 		}, {
-			input:  []string{`$path$`},
-			output: `"$path$"`,
+			input:  []string{`$path$`, `$argument`},
+			output: `"$path$" "$$argument"`,
+		}, {
+			input:  []string{`path`, `Args\nwith\nnewlines`},
```

Comment:
We can't parse the newline with the test framework that's why I removed the newline from the expected output. If the characters after the newline are printed in that form, the escape sequences for the systemd service have worked and especially the newline didn't break the service file.